README.md

```diff
@@ -51,8 +51,8 @@ make superuser
 If you'd like to work with production data and have access, run the following commands:
 
 ```
-make pull-production-data
-make pull-production-media
+fab pull-production-data
+fab pull-production-media
 ```
 
 (you will be prompted to log in to Heroku both times. To log in, hit enter when it asks you and copy and paste the URL it gives you into a browser)
```

Comment:
```suggestion
You will be prompted to log in to Heroku for both commands. To log in, hit enter when it asks you and copy and paste the URL it gives you into a browser. **NOTE:** You will need AWS-CLI installed locally on your machine to run the media command.
```
I think we need a note to remind people that AWS-CLI needs to be installed locally for the `fab pull-production-media` command to work. Something like this maybe?